pkgs/development/libraries/cogl/default.nix

```diff
@@ -35,26 +36,32 @@ stdenv.mkDerivation rec {
 
   configureFlags = [
     "--enable-introspection"
+  ] ++ lib.optionals (!stdenv.isDarwin) [
     "--enable-kms-egl-platform"
     "--enable-wayland-egl-platform"
     "--enable-wayland-egl-server"
   ] ++ lib.optional gstreamerSupport "--enable-cogl-gst"
-  ++ lib.optionals (!stdenv.isDarwin) [ "--enable-gles1" "--enable-gles2" ];
+  ++ lib.optionals (!stdenv.isDarwin) [ "--enable-gles1" "--enable-gles2" ]
+  ++ lib.optionals stdenv.isDarwin [ "--disable-glx" "--without-x" ];
```

Comment:
Similarly for new flags. It will make bisecting easier if case new flags are added in the future.